importAllData

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+set -e
+set -u
+./museum/importObjects -r -p -m
```

Comment:
so the first call refreshes the source data, produces a data report and loads the mapping and the second one runs the import